src/task.js

```diff
@@ -0,0 +1,49 @@
+class Task {
+  constructor(description, index) {
+    this.description = description;
+    this.completed = false;
+    this.index = index;
+  }
+
+  Add = () => {
+    let tasks = [];
+    const storage = JSON.parse(localStorage.getItem('to_do_list'));
+    if (storage !== null) {
+      tasks = storage;
+    }
+    tasks.push({
+      description: this.description,
+      completed: this.completed,
+      index: this.index,
+    });
+    localStorage.setItem('to_do_list', JSON.stringify(tasks));
+  };
```

Comment:
A function for adding a new task (add a new element to the array) is implemented ✔